Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/Search.hs

```diff
@@ -129,16 +133,13 @@ mkIntent origin destination customerLanguage disabilityTag distance duration mbP
                       ]
                 else Nothing,
             customer =
-              if isJust customerLanguage || isJust disabilityTag
-                then
-                  Just $
-                    Search.Customer
-                      { person =
-                          Search.Person
-                            { tags = Search.TG [mkCustomerInfoTags]
-                            }
-                      }
-                else Nothing
+              Just $
```

Comment:
`if isJust customerLanguage || isJust disabilityTag || cancellationDues > 0` can also be used